@xen-orchestra/web/src/components/host/HostPifsTable.vue

```diff
@@ -0,0 +1,257 @@
+              <div v-else-if="column.id === 'status'" class="status">
+                <PifStatus :network="getNetwork(row.value)" />
+              </div>
+              <UiButtonIcon
+                v-else-if="column.id === 'more'"
+                v-tooltip="$t('coming-soon')"
+                :icon="faEllipsis"
+                accent="info"
+                disabled
+                size="small"
+              />
+              <div v-else v-tooltip="{ placement: 'bottom-end' }" class="text-ellipsis">
+                <UiComplexIcon v-if="column.id === 'network'" size="medium" class="icon">
+                  <VtsIcon :icon="faNetworkWired" accent="current" />
+                  <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
+                </UiComplexIcon>
+                {{ column.value }}
+              </div>
+            </td>
+          </tr>
+        </template>
+      </VtsDataTable>
+      <UiTopBottomTable :selected-items="0" :total-items="0" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import PifStatus from '@/components/pif/PifStatus.vue'
+import { useNetworkStore } from '@/stores/xo-rest-api/network.store'
+import { usePifStore } from '@/stores/xo-rest-api/pif.store'
+import type { XoHost } from '@/types/xo/host.type'
+import type { XoPif } from '@/types/xo/pif.type'
+import VtsDataTable from '@core/components/data-table/VtsDataTable.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiActionsTitle from '@core/components/ui/actions-title/UiActionsTitle.vue'
+import UiButton from '@core/components/ui/button/UiButton.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiComplexIcon from '@core/components/ui/complex-icon/UiComplexIcon.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTableActions from '@core/components/ui/table-actions/UiTableActions.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { useRouteQuery } from '@core/composables/route-query.composable'
+import useMultiSelect from '@core/composables/table/multi-select.composable'
+import { useTable } from '@core/composables/table.composable'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import {
+  faAlignLeft,
+  faArrowsRotate,
+  faAt,
+  faCaretDown,
+  faEdit,
+  faEllipsis,
+  faPowerOff,
+  faTrash,
+  faNetworkWired,
+  faCircle,
+  faCheck,
+} from '@fortawesome/free-solid-svg-icons'
+import { noop } from '@vueuse/shared'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { host } = defineProps<{
+  host: XoHost
+}>()
+
+const { get, isReady, hasError } = useNetworkStore().subscribe()
```

Comment:
I think we should use `isReady` and `hasError` from the Pif Store, since we are displaying a list of PIFs.